k9mail/src/main/java/com/fsck/k9/activity/Accounts.java

```diff
@@ -132,7 +132,7 @@
     private FontSizes mFontSizes = K9.getFontSizes();
 
     private MenuItem mRefreshMenuItem;
-    private ActionBar mActionBar;
+    private android.support.v7.app.ActionBar mActionBar;
```

Comment:
Can you remove the old import for `ActionBar` so the unqualified name can be used here?